src/CosmosEditorManager.ts

```diff
@@ -59,7 +59,8 @@ export class CosmosEditorManager {
             }
         }
 
-        const localDocPath = path.join(os.tmpdir(), 'vscode-cosmosdb-editor', fileName);
+        const localFilename = fileName.replace(/[<>:"\/\\|?*]/g, "--");
```

Comment:
nit: I'd prefer to replace with a single hyphen just to keep the length of the file name the same